app/views/users/show.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<div class="user-post">
+   <div class="user-card">
+    
+    <div class="user-image">
+          <img src="<%= @user.photo_link %>" alt="<%= @user.name %> photo" class="img-fluid rounded-circle">
+      </div> 
+   <div class="user-info">
+  <h3><%= @user.name %></h3>
+  <p class="comment">Number of posts: <%= @user.posts.count %></p>
+  </div>
+  </div>
+
+  <div class="user-bio">
+  <p>Bio: <%= @user.bio %></p>
+  </div>
+  
+ <% @user.posts.each_with_index do |post, index| %>
+<div class="allposts-user">
+    <h3>Post <%= index + 1 %></h3>
+      <p class="p-text"><%= post.text %></p>
+      <div class="likes">
+        <p class="post-comment"> comments: <%= post.comments.count %></p>
+        <p class="likes">likes: <%= post.likes.count %></p>
+      </div>
+  </div>
+<% end %>
+
+  <button class="all-posts-button" onclick="window.location.href='<%= user_posts_path(@user) %>'">See all posts</button>
+
+</div>
```

Comment:
- [ ] Kindly maintain the layout of the user show page as presented on the wireframe. The bio and post containers should stretch the width of the page as we on the wireframe layout.

- [ ] The `Number of posts: 1` item should be placed at the bottom right of the user details container

  Your Implementation | Wireframe layout
  :---:|:---:
  ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/8eb2cd40-6b24-43f0-857d-9c91e3771bbb) | ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/89f3ebec-e44e-4f5f-8cc6-e49f399a629e)